stl/inc/execution

```diff
@@ -1216,6 +1245,8 @@ void for_each(_ExPo&&, _FwdIt _First, _FwdIt _Last, _Fn _Func) noexcept /* termi
             }
         }
 
+        _For_each_ivdep(_UFirst, _ULast, _Pass_fn(_Func));
+    } else if constexpr (remove_reference_t<_ExPo>::_Parallelize) {

```

Comment:
I'm confused: isn't this the same as the previous condition? Oh, I think you mean:
```suggestion
    } else if constexpr (remove_reference_t<_ExPo>::_Ivdep) {
```